examples/mlperf/training_submission_v4.1/tinycorp/benchmarks/bert/implementations/tinybox_red/README.md

```diff
@@ -0,0 +1,84 @@
+# 1. Problem
+
+This problem uses BERT for NLP.
+
+## Requirements
+
+Install tinygrad and mlperf-logging from master.
+```
+git clone https://github.com/tinygrad/tinygrad.git
+python3 -m pip install -e ".[mlperf]"
+```
+Also install tqdm, tensorflow and pillow.
```

Comment:
do we need pillow?

Reply:
ye, its imported in `dataloader.py` for resnet. Could move the import though

Reply:
yea move it so we don't need pillow

ideally we also want to use our tqdm but that can be done separately